sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -113,7 +113,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace _internal
       // Saving the value in a local is required before passing it in to Response<T> to avoid
       // compiler optimizations re-ordering the `factoryFn` function call and the RawResponse move.
       T value = factoryFn(*response);
-      return Azure::Response<T>(value, std::move(response));
+      return Azure::Response<T>(std::move(value), std::move(response));
```

Comment:
Not related to this PR

Reply:
sadly, it is.
Since the Page object now contains a raw response, which is a unique_ptr, the current impl failed here trying to copy the value, so I had to updated it